internal/web/container_actions.go

```diff
@@ -0,0 +1,28 @@
+package web
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+)
+
+func (h *handler) containerActions(w http.ResponseWriter, r *http.Request) {
+	action := chi.URLParam(r, "action")
+	id := chi.URLParam(r, "id")
+
+	for _, client := range h.clients {
+		cont, err := client.FindContainer(id)
+
+		if err == nil {
+			if client.ContainerActions(action, cont.ID) != nil {
```

Comment:
it would be better to get the err and log it with `log.Error` since users might be confused why it isn't working. 

Also looking at the UI, it looks like you are showing a toast. But you don't return anything here, is that intentional?

Reply:
Yes, I have added error logs now.

Reply:
Actually i wanted to notify error toast based on the `statusCode` recieved!